src/sys/windows/event.rs

```diff
@@ -6,7 +6,10 @@ pub struct Event {
 
 use miow::iocp::CompletionStatus;
 
-use super::afd;
+use super::afd::{
+    AFD_POLL_ABORT, AFD_POLL_ACCEPT, AFD_POLL_CONNECT_FAIL, AFD_POLL_DISCONNECT, AFD_POLL_RECEIVE,
```

Comment:
take it or leave it (not a bit deal either way): You could drop the `AFD_` prefix from the constants and then reference them as `afd::POLL_ABORT`.

Reply:
I like that, I'll change it.